### ISSUE TITLE
Missing documentation on manipulating sessions

### ISSUE THREAD

**Issue by Aetherus** [balderdashy/sails-docs#445]
Although there is documentation about how to configure session store, I can't find any documentation about how to create, destroy, and access sessions in Sails application. Could you add these informations please?

**Comment by CWyrtzen** [comment]
@rachaelshaw I am wondering if some of these issues are addressed in the sailscasts http://irlnathan.github.io/sailscasts/?

**Comment by mikermcneil** [comment]
@Aetherus please refer to the connect session middleware for details.  However, if you are manually accessing low-level functionality of the session store, you should probably probably switch to using a `Session` model.  Keep in mind, for 99% of use cases, the built-in session store is exactly what you need.

If on the other hand you mean creating/destroying/accessing sessions as in login/logout/signup, etc, I'd encourage you to check out the tutorial I created here: https://github.com/sails101/basic-login

**Comment by mikermcneil** [comment]
oops didn't mean to close

I looked around our docs a bit, and I see what you mean now-- it would make sense to add a top-level section to "Concepts" on "Session", indicating a high-level summary on how the default session support works, and how to use it, e.g.

```
req.session.me = 'some id';
// then some point later on...
var currentUserId = req.session.me;
```

**Comment by sailsbot** [comment]
Thanks for posting, @Aetherus.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
